src/lib/client/adapters/webcontainer/index.js

```diff
@@ -45,12 +51,40 @@ export async function create(base, error, progress, logs) {
 		}
 	});
 
+	/** @type {Record<string, CompilerWarning[]>} */
+	let $warnings;
+	warnings.subscribe((value) => $warnings = value);
```

Comment:
With this there's a dependency loop between the adapter and the output. Can we move the `warnings` store into `tutorial/[slug]/adapter.js` (like the other stores) to decouple this?

Reply:
Thanks for the review. I fixed it!